src/schema/notifications.ts

```diff
@@ -205,187 +215,216 @@ export class ZimbraNotifications {
 	};
 
 	private processContactNotifications = (items: any) => {
-		if (items) {
```

Comment:
Curious to know why have we done this change.

Reply:
This is already handled in `batchProcessItems`. Hence did that.

Reply:
aah okay